docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md

```diff
@@ -0,0 +1,29 @@
+---
+id: call-activity-linking
+title: Call activity linking
+description: Call activity linking
+---
+
+You can use one of the following approaches to link the process to be called by a [call activity](/components/modeler/bpmn/call-activities/call-activities.md).
```

Comment:
@nithinssabu which page should this link display? Maybe I am missing something, but I can't find it.

Reply:
Nevermind, found it: https://docs.camunda.io/docs/components/modeler/bpmn/call-activities/.